versioned_docs/version-8.7/self-managed/operational-guides/update-guide/860-to-870.md

```diff
@@ -124,3 +124,7 @@ For more information, see the [Connectors configuration guide](/self-managed/con
 ### Potential issue leading to stopped processing after update
 
 When updating from `8.6.13` to `8.7.0`, Zeebe processing can stop after the update in some situations, where multi-instance elements are used. This issue is fixed in `8.7.1`. When affected, going to `8.7.1` fully mitigates the issue. There is no risk of data loss.
+
+### Exporter deletion
+
+From version `8.7.12`, exporter configuration removal no longer deletes the exporter from the system. Instead, the exporter enters a blocked state to safeguard against data loss caused by accidental or manual configuration changes. To permanently remove an exporter, use the [Exporters API](/self-managed/zeebe-deployment/operations/management-api.md).
```

Comment:
@christinaausley A similar note must be added to upgrade from 8.7 to 8.8. Could you please advice me where to add such a note?

Reply:
Hi 😄 You are looking for https://docs.camunda.io/docs/next/self-managed/components/components-upgrade/870-to-880/?

Reply:
Thanks. I added it there.